config/locales/custom/verification.yml

```
@@ -33,10 +35,12 @@ ro:
         verify_residence: Verifică
         help_needed: |
           Dacă întâmpini probleme cu procesul de verificare, contactează-ne: %{email}
+        support_email: contact@primaria-dumbravita.ro
         document_number_help_text: "<strong>Exemplu CNP: 2900411192534</strong>"
         document_number: Cod Numeric Personal (CNP)
         accept_terms_adult: Declar pe proprie răspundere că am împlinit 18 ani
         accept_terms_resident: Declar pe proprie răspundere că am reședinta temporară sau permanentă în %{residence}
+        city_name: Dumbrăvița
         accept_terms_terms: Accept %{terms_url} și %{privacy_url}
         privacy: politica de confidențialitate
         explanation: Folosim CNP-ul dumneavoastră doar pentru a valida faptul că sunteți rezident în comuna Dumbrăvița, conform %{rules_url}.
```

Maybe we can reduce the number of places where the city name must be changed :thinking:.

```suggestion
        explanation: Folosim CNP-ul dumneavoastră doar pentru a valida faptul că sunteți rezident în comuna %{residence}, conform %{rules_url}.
```

Of course, we'd need to change the view and pass the parameter from there.